playbooks/roles/os_temps/tasks/setup_containers.yml

```diff
@@ -18,16 +19,27 @@
     msg: "{{ item.path }}"
   with_items: "{{ os_templates.files }}"
 
+- name: "Filter the templates"
+  include_tasks: "filter_templates.yml"
+  with_items: "{{ os_templates.files }}"
```

Comment:
you'll have to change here too

Reply:
That one is using the original registered os_templates variable from the command defined on line 15, and then filling the empty os_template_files list with templates that pass the filtering.